### PR TITLE
Feat auth

### PR DESCRIPTION
**Pull request by rahulyadav139**

### CODE REVIEW

File: src/components/authorization/SignupForm.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ const SignupForm = props => {
 
     if (error) return;
 
-    dispatch(AuthActions.loginHandler(data));
+    // dispatch(AuthActions.loginHandler(data));
```
Comment (HarshnaNHaswani) [inline src/components/authorization/SignupForm.js]: please remove this if not needed

**Review by krate27 — APPROVED** [review]
Things look good. Liked the way you have used object destructuring and raised a small pr for readability. 

Good to merge from my side.

File: src/components/authorization/SignupForm.js [hunk 2]
```diff
@@ -89,18 +99,24 @@ const SignupForm = props => {
...
       !firstNameIsValid ||
       !lastNameIsValid ||
+      !usernameIsValid ||
       !emailIsValid ||
       !passwordIsValid ||
       !confirmPasswordIsValid
     ) {
       firstNameIsTouched(true);
       lastNameIsTouched(true);
+      usernameIsTouched(true);
```
Comment (HarshnaNHaswani) [inline src/components/authorization/SignupForm.js:109]: what is the purpose of this?

**Review by HarshnaNHaswani — COMMENTED** [review]
Looks good to merge, good naming conventions and clean, formatted code structure

**Pull request merged**